website/docs/cdktf/concepts/stacks.html.md

```diff
@@ -9,6 +9,8 @@ description: "Use stacks to specify separate collections of infrastructure for d
 
 -> **Note:** CDK for Terraform is currently in [beta](/docs/cdktf/index.html#project-maturity-and-production-readiness).
 
+> **Hands-on:** Try the [Deploy Applications with CDK for Terraform](https://learn.hashicorp.com/tutorials/terraform/cdktf-applications?in=terraform/cdktf&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
```

Comment:
I love this - thank you for adding! I wonder - can we move it to below the description of what stacks is? Just right above the single stack header?

Also! I just realized that the headings seem to be off here - could you change all three of these on this page to be H2s?

Reply:
Fixed them @laurapacilio, please confirm :)